src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/fluid.py

```diff
@@ -62,7 +62,6 @@ def __init__(
             )
 
             self.standard_conditions_density = _neqsim_fluid_at_standard_conditions.density
-            self.molar_mass_kg_per_mol = _neqsim_fluid_at_standard_conditions.molar_mass
```

Comment:
not needed?

Reply:
Could not find it to be used anywhere. So I think so.